terminus-ui/input/testing/src/test-helpers.ts

```diff
@@ -44,7 +44,7 @@ export function getInputElement(fixture: ComponentFixture<any>, index = 0): HTML
   if (!instance.inputElement) {
     throw new Error(`'getInputElement' found no input elements at index '${index}'`);
   }
-  return instance.inputElement.nativeElement as HTMLInputElement;
+  return instance.inputElement.nativeElement;
```

Comment:
it's removed because that implicitly returns `HTMLInputElement`? I do sometimes spell it out just to be explicit.

Reply:
Yep, our linter fix has been pulling these out lately.

Originally we used it because I _think_ we were losing some typing strength due to Angular or TS. But it seems whoever was weak has improved, so our linter is removing unnecessary types.